templates/config/barong/seeds.yml.erb

```diff
@@ -0,0 +1,179 @@
+# List of Barong administrators.
+#
+# Other parameters that you can set here can be found in Account,
+# Profile and Phone models (e.g. app/models/account.rb...); all the
+# model properties are listed at the very bottom of this models files.
+#
+# Note: email is the only required field here.
+users:
+  - email: admin@barong.io
+    password: 0lDHd9ufs9t@
+    role: admin
+    state: active
+    level: 3
+  - email: john@barong.io
+    password: Am8icnzEI3d!
+    role: member
+    state: active
+    level: 3
+
+levels:
+  - key: email
+    value: verified
+    description: "User clicked on the confirmation link"
+  - key: phone
+    value: verified
+    description: "User entered a valid code from sms"
+  - key: document
+    value: verified
+    description: "User personal documents have been verified"
+
+permissions:
+# SUPER ADMIN has an access to the whole system without any limits
+# so we can just grant him access to /api/v2/*
+ - { role: superadmin, verb: all, path: api/v2, action: accept }
+ - { role: superadmin, verb: post, path: api/v2/peatio/market/orders, action: drop }
```

Comment:
finex condition is missing here

Reply:
It was before, I think superadmin should not trade

Reply:
I would keep allowing him by default, this is useful for testing, then each platform can still block it.